package.json

```diff
@@ -118,5 +118,13 @@
     "help": "yamscripts help",
     "yamscripts": "yamscripts compile",
     "__": "# NOTE: THESE SCRIPTS ARE COMPILED!!! EDIT yamscripts.yml instead!!!"
-  }
+  },
+  "preferences": [
+    {
+      "name": "performanceLog",
+      "title": "Enable Performance Logging",
+      "type": "bool",
+      "value": true
```

Comment:
we want the default to be `false`. Enable in test mode in the `before` statement.